wordpress-mobile/WordPress-iOS
# Issue/5245 migrate sharing services to nsurlsession

Refs #5245 

To test:
 - Login with a WP.com account
 - Go To Blog->Sharing
 - Test if connections show correctly
 - Try to connect to services and see if it works correctly
 - Delete connections after and check if they work.

Needs review: @aerych
